openedx/core/djangoapps/schedules/utils.py

```diff
@@ -57,7 +57,7 @@ def reset_self_paced_schedule(user, course_key, use_availability_date=False):
                 enrollment__course__self_paced=True,
             )
         except Schedule.DoesNotExist:
-            return
+            raise Schedule.DoesNotExist
```

Comment:
You can simply do `raise` with no arguments to re-raise the current exception.

Reply:
ty